<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestingWidget</class>
 <widget class="QWidget" name="TestingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="leftLayout">
     <item>
      <widget class="QLabel" name="lblTestingWindow">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Testing window:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="buddy">
        <cstring>m_testingTextBrowser</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="m_testingTextBrowser">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="m_frmConfirm">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QStackedWidget" name="m_swInputData">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="m_wChoice">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="lblChoice">
              <property name="text">
               <string>Choose your answer:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="m_cbIndex0">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="m_cbIndex1">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="m_cbIndex2">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="m_cbIndex3">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="m_cbIndex4">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="m_cbIndex5">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="m_cbIndex6">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="m_cbIndex7">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="m_cbIndex8">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="m_wIntegerInput">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="lblInteger">
              <property name="text">
               <string>Enter an integer number:</string>
              </property>
              <property name="buddy">
               <cstring>m_sbInteger</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="m_sbInteger"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="m_wDoubleInput">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="lblDouble">
              <property name="text">
               <string>Enter a fractional number:</string>
              </property>
              <property name="buddy">
               <cstring>m_sbInteger</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m_sbDouble"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="confirmLayout">
          <item>
           <widget class="QPushButton" name="m_pbConfirm">
            <property name="text">
             <string>Confirm answer</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="rightLayout">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="m_gbTopics">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Choosing a topic</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="m_rbAllTopics">
          <property name="text">
           <string>Everything</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_rbDiscreteMathematics">
          <property name="text">
           <string>Discrete mathematics</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_rbHashFunction">
          <property name="text">
           <string>Hash function</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_rbHashTable">
          <property name="text">
           <string>Hash table</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_rbAlgorithm">
          <property name="text">
           <string>Algorithm</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="m_gbDifficulty">
       <property name="title">
        <string>Choice of difficulty</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="m_rbAllDifficulties">
          <property name="text">
           <string>Everything</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_rbEasy">
          <property name="text">
           <string>Easy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_rbMedium">
          <property name="text">
           <string>Medium</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_rbHard">
          <property name="text">
           <string>Hard</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pbStart">
       <property name="text">
        <string>Start testing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pbStop">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save &amp;&amp; Stop testing</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_testingTextBrowser</tabstop>
  <tabstop>m_pbStart</tabstop>
  <tabstop>m_pbStop</tabstop>
  <tabstop>m_sbInteger</tabstop>
  <tabstop>m_pbConfirm</tabstop>
  <tabstop>m_rbAllTopics</tabstop>
  <tabstop>m_rbDiscreteMathematics</tabstop>
  <tabstop>m_rbHashFunction</tabstop>
  <tabstop>m_rbHashTable</tabstop>
  <tabstop>m_rbAlgorithm</tabstop>
  <tabstop>m_rbAllDifficulties</tabstop>
  <tabstop>m_rbEasy</tabstop>
  <tabstop>m_rbMedium</tabstop>
  <tabstop>m_rbHard</tabstop>
  <tabstop>m_sbDouble</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
